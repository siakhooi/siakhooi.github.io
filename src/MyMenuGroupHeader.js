import React from 'react'
import PropTypes from 'prop-types'

MyMenuGroupHeader.propTypes = {
  children: PropTypes.string
}

function MyMenuGroupHeader (props) {
  return (
    <h2 className="MyMenuGroupHeader">{props.children}</h2>
  )
}
export default MyMenuGroupHeader
