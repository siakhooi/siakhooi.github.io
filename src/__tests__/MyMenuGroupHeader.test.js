import React from 'react'
import renderer from 'react-test-renderer'
import MyMenuGroupHeader from '../MyMenuGroupHeader'

it('MyMenuGroupHeader/1', () => {
  const component =
    renderer.create(
      <MyMenuGroupHeader>Hello Data</MyMenuGroupHeader>
    )
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
