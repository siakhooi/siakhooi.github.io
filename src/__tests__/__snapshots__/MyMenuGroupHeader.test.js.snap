// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MyMenuGroupHeader/1 1`] = `
<h2
  className="MyMenuGroupHeader"
>
  Hello Data
</h2>
`;
